src/plugins/kibana/public/settings/sections/indices/add_data_steps/upload_data_step/upload_data_step.js

```diff
@@ -28,7 +28,7 @@ modules.get('apps/settings')
           _.forEach(res.data, (response) => {
             this.created += response.created;
             this.formattedErrors = this.formattedErrors.concat(_.map(_.get(response, 'errors.index'), (doc) => {
-              return `${doc._id.split('-', 1)[0].replace('L', 'Line ').trim()}: ${doc.error.type} - ${doc.error.reason}`;
+              return `Line ${doc._id.substr(doc._id.lastIndexOf(':') + 1)}: ${doc.error.type} - ${doc.error.reason}`;
```

Comment:
Is it worth pulling this logic into a function? Especially since the format of the data has changed now during the development of this feature?